cdisc_rules_engine/utilities/data_processor.py

```diff
@@ -327,6 +375,29 @@ def _get_columns_by_meddra_variable_name(
         }
         return variable_pair_map.get(params.target)
 
+    def _get_metadata_for_class_and_domain(
+        self, dataset_class: str, domain: str, standard: str, standard_version: str
+    ) -> Optional[dict]:
+        """
+        Gets metadata for the given class and domain from cache.
+        The cache stores CDISC Library metadata.
+        """
+        standard_details: dict = (
```

Comment:
Since this is a standalone application now and the cache may not always be populated do we think it's a good idea to start making requests to the library if the data is not found in cache? I don't think that would have to be updated in this PR, but an issue should be created for it if we want to start doing that